docs/src/user/library/evc_results.rst

```diff
@@ -1,7 +1,7 @@
 Iterative Results with EVC
 --------------------------
 
-When using the experiment version control (described `here <user/evc>`_),
+When using the experiment version control (described :doc:`here <../../user/evc>`),
```

Comment:
Thanks for the fix! Can you try with `/user/evc` instead?

Reply:
Great idea! I remember trying it but it didn't work at the time. In hindsight, it was probably a small syntax error. I tried again and it worked. It's much better this way 🙏.